snuba/migrations/migrate.py

```diff
@@ -57,28 +57,40 @@ def run() -> None:
     for storage_key in STORAGES_TO_MIGRATE:
         run_storage(storage_key)
 
+    # Additional schemas to be migrated
+    from snuba.datasets.storages.outcomes import (
+        materialized_view_schema as outcomes_mv_schema,
+    )
+    from snuba.datasets.storages.sessions import (
+        materialized_view_schema as sessions_mv_schema,
+    )
+
+    ADDITIONAL_SCHEMAS_TO_MIGRATE = [
+        (StorageSetKey.OUTCOMES, outcomes_mv_schema),
+        (StorageSetKey.SESSIONS, sessions_mv_schema),
+    ]
+
+    for storage_set, schema in ADDITIONAL_SCHEMAS_TO_MIGRATE:
+        conn = get_cluster(storage_set).get_query_connection(
+            ClickhouseClientSettings.MIGRATE
+        )
+        conn.execute(schema.get_local_table_definition().statement)
```

Comment:
This is going to be needed as long as we do not fully use the new migration system and there it won't be needed, is that correct ?

Reply:
Also isn't this executing migrations in a different order than before (before it was write schema, intermediate schema, read schema). Is this change intentional ?

Reply:
Yeah that's right. I moved it over here so it can easily be removed and storage schemas will be cleaner. The order before was read, write, intermediate (as per get_unique_schemas) so it hasn't changed. In any case, read and write were always the same, and intermediate is always the materialized view which must be last.